ee/clickhouse/views/test/test_clickhouse_groups.py

```diff
@@ -54,35 +48,29 @@ def test_groups_list(self):
             },
         )
 
-    def test_groups_list_pagination(self):
```

Comment:
Q: Why remove the pagination test? New test for the new way we're doing pagination?

Reply:
Previously we were hand-rolling pagination but now we are using standard methods from the Django framework. I didn't find it useful to be testing that django is doing the right thing. :)

Reply:
Makes sense.